tsdb/record/record.go

```diff
@@ -44,10 +44,10 @@ const (
 	Tombstones Type = 3
 	// Exemplars is used to match WAL records of type Exemplars.
 	Exemplars Type = 4
-	// Histograms is used to match WAL records of type Histograms.
-	Histograms Type = 5
 	// Metadata is used to match WAL records of type Metadata.
 	Metadata Type = 6
+	// Histograms is used to match WAL records of type Histograms.
+	Histograms Type = 7
```

Comment:
Why this change?

Reply:
Oh I forgot to mention. So the backstory is that the out of order support brings in a new record type that is already using the number 5. Given it is in production at places already, I think it is better to change this one here since native histograms are not anywhere (except one instance in the world).